.circleci/config.yml

```diff
@@ -1063,31 +1072,16 @@ workflows:
             - checkout
 
       - clojure:
-          name: be-tests-<< matrix.edition >>
-          requires:
-            - be-deps
-          e: java-8
-          clojure-args: -X:dev:ci:test
-          skip-when-no-change: true
-          <<: *Matrix
-
-      - clojure:
-          name: be-tests-java-11-<< matrix.edition >>
-          requires:
-            - be-deps
-          e: java-11
-          clojure-args: -X:dev:ci:test
-          skip-when-no-change: true
-          <<: *Matrix
-
-      - clojure:
-          name: be-tests-java-16-<< matrix.edition >>
+          matrix:
+            parameters:
+              edition: ["ee", "oss"]
+              java-version: ["java-8", "java-11", "java-16"]
```

Comment:
Should this be bumped to Java 17 instead?
It's the most recent LTS, released back in Sep 2021.

Reply:
just saw this comment: I have that ready on the new images branch, we definitely have to switch to java 17